cmd/controller-manager/main.go

```diff
@@ -134,20 +159,131 @@ func main() {
 	setupLog.Info("Got config for WDS", "name", wdsName)
 	wdsRestConfig = wdsClientLimits.LimitConfig(wdsRestConfig)
 
-	// get the config for ITS
+	// get the config for ITS with retry for resiliency
 	setupLog.Info("Getting config for ITS")
-	itsRestConfig, itsName, err := ctrlutil.GetITSKubeconfig(setupLog, itsName)
```

Comment:
Note that there is already a retry loop in `ctrlutil.GetITSKubeconfig`. I think that it would be better to extend the retrying in that func than to write a retry loop over every call site of that func.